src/main/java/org/broadinstitute/dsde/consent/ontology/datause/services/TextTranslationServiceImpl.java

```diff
@@ -69,11 +76,14 @@ public class TextTranslationServiceImpl implements TextTranslationService {
 
     private static final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("MM/dd/yyyy");
 
-    private AutocompleteService autocompleteService;
+    private final AutocompleteService autocompleteService;
+
+  private final GCSStore gcsStore;
 
     @Inject
-    public TextTranslationServiceImpl(AutocompleteService autocompleteService) {
+    public TextTranslationServiceImpl(AutocompleteService autocompleteService, GCSStore gcsStore, StoreOntologyService storageService) {
```

Comment:
`storageService` is unused:
```suggestion
    public TextTranslationServiceImpl(AutocompleteService autocompleteService, GCSStore gcsStore) {
```